modules/ROOT/pages/redelivery-policy.adoc

```diff
@@ -0,0 +1,63 @@
+= Redelivery Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, redelivery, policy
+
+A redelivery policy is a filter added to Mule sources, such as modules and connectors, to prevent the dispatch of messages known to cause errors.
```

Comment:
sources are not modules or connectors. Like we discussed via Slack, some message sources in connectors (and not modules) provide a way to set up a redelivery policy.